docs/user/authentication-settings-guide.md

```diff
@@ -14,7 +14,7 @@ JAAS Authenticator will use JAAS login modules.
         className: io.ballerina.messaging.broker.auth.authentication.authenticator.JaasAuthenticator
         # Optional properties
         properties:
-         loginModule: io.ballerina.messaging.broker.auth.authentication.jaas.UserStoreLoginModule
+         loginModule: io.ballerina.messaging.broker.auth.authentication.jaas.DefaultJaasLoginModule
```

Comment:
Can we call this something like `FileBaseJaasLoginModule`?

Reply:
ok